torch/csrc/distributed/rpc/tensorpipe_agent.h

```diff
@@ -91,11 +91,6 @@ class TensorPipeAgent : public RpcAgent {
   std::string createUniqueShmAddr();
 #endif
 
-  // Retrieve IP address for a given network device for corss-hosts
-  // to set up tensorpipe connection. For now we default the device
-  // name eth0.
-  static std::string getDefaultIPAddress();
```

Comment:
Which PR deleted its implementation in the .cpp file?

Reply:
It was in #39184, but I had forgotten to remove the declaration from the header.